agreements/ddl.sql

```diff
@@ -57,6 +57,25 @@ CREATE TABLE lot_sectors (
 
 CREATE INDEX LOT_SECTORS_IDX1 on LOT_SECTORS (lot_id,sector_code);
 
+CREATE TABLE route_to_market (
+  route_to_market_name              VARCHAR(50) PRIMARY KEY,
+  route_to_market_description       VARCHAR(2000) NOT NULL
+);
+
+CREATE TABLE lot_route_to_market (
+  lot_id                            INTEGER PRIMARY KEY,
+  route_to_market_name              VARCHAR(50) PRIMARY KEY,
+  start_date                        DATE NOT NULL,
+  end_date                          DATE NOT NULL,
+  location                          VARCHAR(20), -- For example, Regional or National  
+  lot_minimum_value                 NUMERIC(18,4),         
+  lot_contract_minimum_weeks        SMALLINT, 
+  lot_contract_maximum_weeks        SMALLINT,
+  lot_procurement_minimum_quantity  SMALLINT,
+  lot_procurement_maximum_quantity  SMALLINT

```

Comment:
I think there is a bit of a disconnect here with Dave's API.

The only field I think we require for Phase 1 is `buyingMethods`, and also potentially `buyingSystemURL` (which I think are missing). These help to inform the display on the results page of a Guided Match journey (text and buttons).

Also in the API - quantity and contract length are a composite of unit + length, not a single value. We could infer units for length (weeks) - but seems like a bit of a mismatch. However - as mentioned I don't think these other fields are essential for Phase 1 (from current understanding).

Reply:
By buying methoid do you mean, MarketPlace, Further Competition, Direct Award and Auction. If so they are the records in route_to_method. I did forget in my haste to add the buying method. Yes when I was creating the table I did have contract length uom but I thourght that was overkill and just rationalised it and was going to talk to you guys today. I can put them them back in.